cmd/admin-replicate-update.go

```diff
@@ -40,13 +40,13 @@ var adminReplicateEditFlags = []cli.Flag{
 	},
 }
 
-var adminReplicateEditCmd = cli.Command{
-	Name:         "edit",
-	Usage:        "edit endpoint of site participating in cluster replication",
-	Action:       mainAdminReplicateEdit,
+var adminReplicateUpdateCmd = cli.Command{
+	Name:         "update",
```

Comment:
```suggestion
	Name:         "update",
```

Keep `edit` for backward compatibility reasons? - and update the examples to point to the new command.